src/fetch.js

```diff
@@ -0,0 +1,17 @@
+const fetchStarWarsData = async (type) => {
+  const response = await fetch("https://swapi.dev/api/" + type).catch((err) => {
```

Comment:
You can put everything in the try...catch and don't mix the async/await with .catch

But to be honest, I'm not sure is this just a fashion(to not mix async/await with .catch) or the best practice.